dataactbroker/handlers/fileHandler.py

```diff
@@ -1577,4 +1577,19 @@ def fabs_derivations(obj):
     federal_action_obligation = obj['federal_action_obligation'] or 0
     non_federal_funding_amount = obj['non_federal_funding_amount'] or 0
     obj['total_funding_amount'] = federal_action_obligation + non_federal_funding_amount
+
+    # deriving cfda_title from program_title in cfda_program table
+    sess = GlobalDB.db().session
+
+    try:
+        cfda_title = sess.query(CFDAProgram).filter_by(program_number=obj['cfda_number']).one()
+    except:
+        print('CFDA Title not Found')
```

Comment:
This shouldn't be a print statement. You should use the logger.exception() or logger.debug() or logger.info() function (there's an example in this file somewhere) so it actually gets printed to our logs. Choose the one that you think is most relevant (probably exception but if you think differently then use a different one)

Also, maybe include which cfda_number it wasn't found for ("CFDA Title not found for CFDA Number X") (found shouldn't be capitalized)